src/odemis/driver/piezomotor.py

```diff
@@ -829,6 +830,8 @@ def _sendCommand(self, cmd):
                                   "restarted serial connection.")
             logging.debug("Received response %s", to_str_escape(resp))
 
+            if not resp.endswith(EOL):
+                raise IOError("Response %s does not ends with %s, it's due to a timeout" % (to_str_escape(resp), EOL))
```

Comment:
"%s" % EOL gives quite an odd output `"b'\r'"`. I'd just slightly rephrase this message and hard-code EOL:
"Timeout: no EOL received after response '%s'"